bindings/rust/src/lib.rs

```diff
@@ -14,13 +14,11 @@ pub fn validate<T: AsRef<[u8]>>(input: T) -> bool {
 }
 
 /// A parsed and validated WebAssembly 1.0 module.
-pub struct Module {
-    ptr: NonNull<sys::FizzyModule>,
-}
+pub struct Module(*const sys::FizzyModule);
```

Comment:
Why can't it be done like that from the beginning?

Reply:
I'll squash those commits in, just didn't do it yet.

Reply:
Sounds good.